### PR TITLE
resource/alicloud_db_instance: adjust_instance_creation_port

### PR DESCRIPTION
**Pull request by zwc01090043**

### CODE REVIEW

File: alicloud/resource_alicloud_db_instance.go [hunk 1]
```diff
@@ -1214,15 +1214,7 @@ func resourceAliCloudDBInstanceUpdate(d *schema.ResourceData, meta interface{})
 		"RegionId":     client.RegionId,
 		"SourceIp":     client.SourceIp,
 	}
-	if d.HasChange("port") {
-		connectUpdate = true
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_db_instance.go:1218]: 为啥删了

File: website/docs/r/db_instance.html.markdown [hunk 2]
```diff
@@ -662,7 +662,7 @@ The following arguments are supported:
   - Disable
 
 -> **NOTE:** This parameter only takes effect when the StorageAutoScale parameter is set to Enable.
-* `storage_threshold` - (Optional, Available in 1.129.0+)The trigger threshold (percentage) for automatic storage space expansion.
+* `storage_threshold` - (Optional, Available in 1.129.0+)The threshold in percentage based on which an automatic storage expansion is triggered. If the available storage reaches the threshold, ApsaraDB RDS increases the storage capacity of the instance.
```
Comment (ChenHanZhang) [inline website/docs/r/db_instance.html.markdown:665]: Available since 1.129.0

Comment (zwc01090043) [reply website/docs/r/db_instance.html.markdown]: 已修改

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**